okd/src/use_okd_assets.sh

```diff
@@ -0,0 +1,26 @@
+#/bin/bash
+
+
+# replace microshift assets to Upsteam from OKD
+okd_url=$1
+okd_releaseTag=$2
+
+
+[[ -n $okd_url  && -n $okd_releaseTag ]] || { echo missing vars ;  exit 2 ; }
+
+
+# replace Microshift images with upstream (from OKD release)
+for op in $(cat assets/release/release-x86_64.json  | jq -e -r  '.images | keys []') 
```

Comment:
The way this script is written, it assumes to be executed from a certain location in the tree. Let's make it more robust please.